dbos/_migration.py

```diff
@@ -14,41 +14,46 @@ def ensure_dbos_schema(engine: sa.Engine) -> None:
         # Check if dbos schema exists
         schema_result = conn.execute(
             sa.text(
-                "SELECT schema_name FROM information_schema.schemata WHERE schema_name = 'dbos'"
-            )
+                "SELECT schema_name FROM information_schema.schemata WHERE schema_name = :schema"
+            ),
+            {"schema": schema},
         )
         schema_exists = schema_result.fetchone() is not None
 
         # Create schema if it doesn't exist
         if not schema_exists:
-            conn.execute(sa.text("CREATE SCHEMA dbos"))
+            conn.execute(sa.text(f'CREATE SCHEMA "{schema}"'))
 
         # Check if dbos_migrations table exists
         table_result = conn.execute(
             sa.text(
-                "SELECT table_name FROM information_schema.tables WHERE table_schema = 'dbos' AND table_name = 'dbos_migrations'"
-            )
+                "SELECT table_name FROM information_schema.tables WHERE table_schema = :schema AND table_name = 'dbos_migrations'"
+            ),
+            {"schema": schema},
         )
         table_exists = table_result.fetchone() is not None
 
         if not table_exists:
             conn.execute(
                 sa.text(
-                    "CREATE TABLE dbos.dbos_migrations (version BIGINT NOT NULL PRIMARY KEY)"
+                    f'CREATE TABLE "{schema}".dbos_migrations (version BIGINT NOT NULL PRIMARY KEY)'
```

Comment:
does `sa.text` makes a prepared statement out of the query? If not we might want to sanitize the schema